manifests/params.pp

```diff
@@ -13,25 +13,37 @@
         $service_file_location   = '/usr/lib/systemd/system/jira.service'
         $service_file_template   = 'jira/jira.service.erb'
         $service_lockfile        = '/var/lock/subsys/jira'
+        $service_provider        = 'systemd'
       } elsif $::operatingsystemmajrelease == '6' or $::operatingsystem == 'Amazon'{
         $json_packages           = [ 'rubygem-json', 'ruby-json' ]
         $service_file_location   = '/etc/init.d/jira'
         $service_file_template   = 'jira/jira.initscript.erb'
         $service_lockfile        = '/var/lock/subsys/jira'
+        $service_provider        = 'init'
       } else {
         fail("\"${module_name}\" provides no service parameters
             for \"${::osfamily}\" - \"${::operatingsystemmajrelease}\"")
       }
     } /Debian/: {
+      if $::operatingsystemmajrelease == '8' {
```

Comment:
Hmm OK. So, the big guns?

    if versioncmp($::operatingsystemmajrelease, '8') > -1

Thoughts?